examples/vacanza.py

```
@@ -0,0 +1,153 @@
+def get_supported_categories(country_code):
+    """Return supported holiday categories for the given country code."""
+    try:
+        country_class = holidays.country_holidays(country_code)
+        if hasattr(country_class, "supported_categories"):
+            return country_class.supported_categories
+        return (PUBLIC,)
+    except Exception:
+        return (PUBLIC,)
+
+
+def generate_ics(country_code, years, public_only=False):
```

Ruff (0.15.6)

[warning] Boolean default positional argument in function definition

(FBT002)